incubator/java-openliberty/templates/kafka/README.md

```diff
@@ -0,0 +1,76 @@
+# Kafka template for Open Liberty
+
+This is a template for the development of Liberty applications that can connect to Kafka using MicroProfile Reactive Messaging. There is a simple StarterApplication that enables basic production and consumption of events. The instructions for getting this up and running are below. As part of this, we have included a `docker-compose.yaml` file which will start a local Kafka and Zookeeper. 
```

Comment:
For clarity, to avoid "this is" and "there is" .....

This template can be used to develop Liberty applications that connect to Kafka by using MicroProfile Reactive messaging. A simple `StarterApplication` is included that ..... 

Please remove "The instructions for getting this up and running are below."  It isn't needed. You could move the information about the docker-compose.yaml` under the **Getting started with the `StarterApplication`. For example, "These instructions include a `docker-compose.yaml` file, which starts a local Kafka and Zookeeper (microservice?).